markdown/ranger/ranger-integration-config.html.md.erb

```diff
@@ -70,9 +75,25 @@ The following procedures describe each configuration activity.
     gpadmin@master$ ./enable-ranger-plugin.sh -r ranger_host:6080 -u admin -p admin -h hawq_master:5432 -w gpadmin -q gpadmin
     ```
     
+    ***Note*** You can also enter the short form of the command: `./enable-ranger-plugin.sh -r` and the script will prompt you for entries. 
+    
     When the script completes, the default HAWQ service definition is registered in the Ranger Admin UI. This service definition is named `hawq`.
 
-6. Edit the `pg_hba.conf` file on the HAWQ master node to configure HAWQ access for \<hawq_user\> on the \<ranger-admin-node\>. For example, you would add an entry similar to the following for the example `enable-ranger-plugin.sh` call above:
+6. Locate the `pg_hba.conf` file on the HAWQ master node:
+ 
+    ``` bash
+    $ hawq config --show hawq_master_directory
+     GUC		: hawq_master_directory
+     Value		: /data/hawq/master
+     $ ls /data/hawq/master
```

Comment:
will listing the directory contents help the user?  i find it kind of distracting.